multimodal/src/autogluon/multimodal/learners/base.py

```diff
@@ -21,6 +21,7 @@
 from lightning.pytorch.strategies import DeepSpeedStrategy
 from omegaconf import DictConfig, OmegaConf
 from packaging import version
+from safetensors.torch import load_file, save_file
 from torch import nn
```

Comment:
Thanks for the PR!

We should also remove `--skip B614` here once this PR is ready
https://github.com/autogluon/autogluon/blob/master/.github/workflow_scripts/lint_check.sh#L25

Reply:
Several AutoMM tests failed due to an inconsistency in managing shared tensors between `torch.save` and `save_file` from `safetensors.torch`. It appears that no solution exists for this issue yet, particularly when `top_k_average_method` is set to `best`. Consequently, I must abandon this PR and annotate the code with `#nosec B614` to address the bandit warning and prevent future use of torch.save() in externally exposed APIs.

Reply:
Makes sense, this looks like a reasonable way to address the issue.